.travis.rosinstall

```diff
@@ -0,0 +1,4 @@
+- git:
+    local-name: ros_control_boilerplate
+    uri: https://github.com/PickNikRobotics/ros_control_boilerplate.git
```

Comment:
create a mojin fork to freeze that version

Reply:
better depend on a certain version/tag

Reply:
I still prefer to not depend on some external repositories where the admins could force-push, rename the repo, delete tags, ...
So even if we depend on a certain version/tag I'd like to have github.com/mojin-robotics in there